### ISSUE TITLE
Installing the NuGet package results in a fatal error

### ISSUE THREAD

**Issue by mohamed-abdelghaffar214** [Microsoft/Win2D#946]
I installed the NuGet package for Win2D but it happened to break my project for some reason. I'm not an expert, so I was trying to add the DDSTextureLoader to my WPF project, and stack overflow suggested Win2D since I didn't need any 3D to warrant full DirectX integration. I'm hesitant to call this a bug, it could just be my fault, but I'd appreciate the help regardless. Here is the error:
```
A numeric comparison was attempted on "$(TargetPlatformVersion)" that evaluates to "" instead of a number, in condition "'$(TargetPlatformVersion)' < '10.0.18362.0'".
C:\Users\%Username%\.nuget\packages\microsoft.windowsappssdk\1.4.231115000\buildTransitive\Microsoft.InteractiveExperiences.common.targets
```
I went to that path and all it had that was related was these couple of lines:
```
<!-- Set the UAP metadata to reference based on the target version -->
<PropertyGroup>
  <Ixp-UAPTargetVersion Condition="'$(TargetPlatformVersion)' &lt; '10.0.18362.0'">uap10.0.17763</Ixp-UAPTargetVersion>
  <Ixp-UAPTargetVersion Condition="'$(Ixp-UAPTargetVersion)' == ''">uap10.0.18362</Ixp-UAPTargetVersion>
</PropertyGroup>
```
Is it supposed to be this way? Help!

**Comment by getrou** [comment]
Hello, could you give me a bit more information about your app?

1. Is your app a UWP/UAP project or a Win32/Desktop Application? 
2. What version of Win2D are you using?

**Comment by mohamed-abdelghaffar214** [comment]
It's a WPF Desktop application. I ended up using a different API, but this is still a problem that should be solved. My guess would be it wasn't made to support backwards compatibility for other versions of Windows SDK. Win2D's version number 1.1.1, supposedly the latest stable version. When I click install within Visual Studio's UI, it shows me additional packages needed for the main package to be installed. Here is a screenshot:

![image](https://github.com/microsoft/Win2D/assets/52886329/fe07a050-867b-4bd1-8d22-d146c0aa4b0a)

Restarting VS didn't help unlike what stack overflow suggested. It remained broken even after I manually removed the Win2D project link, but I solved it by specifying the latest Windows SDK version `net8.0-windows10.0.22621.0` instead of just `net8.0-windows` in the `<TargetFramework>` tag, since I had both `10.0.22621.0,` and `10.0.18362.0` installed here:

![image](https://github.com/microsoft/Win2D/assets/52886329/39532c03-a5fc-470d-bfb4-51416b06a78a)

I'll leave it to you to decide whether the issue needs solving on your end, or just to be closed.
